src/components/overview/chart/index.tsx

```diff
@@ -0,0 +1,90 @@
+import CustomizedDot from '../customizedDot';
+import { colors } from '../../../theme';
+import { useState } from 'react';
+import {
+  Area,
+  AreaChart,
+  CartesianGrid,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { data, gradients, yDomain, yTicks } from './helper';
+import { tick, xTickPadding } from './styled';
+
+const Chart = () => {
+  const currentDate = new Date();
+  const currentHour = currentDate.getHours();
+  const [isTodayHovered, setIsTodayHovered] = useState(false);
+  const [isYesterdayHovered, setIsYesterdayHovered] = useState(false);
+
+  const filteredData = data.map((entry) => ({
```

Comment:
How we use this filteredData? can we set necessary data to the `data` directly instead of mapping

Reply:
fix: Yesterday graph is fully showed. Today calculations moved to a separate function. FiltredData renamed to currentData